pkg/providers/imagefamily/nodeimage_test.go

```diff
@@ -62,12 +67,17 @@ func getExpectedTestCIGImages(imageFamily string, version string) []imagefamily.
 	return nodeImages
 }
 
-func getExpectedTestSIGImages(imageFamily string, version string) []imagefamily.NodeImage {
+//nolint:unparam // might always be using the same version in test, but could change in the future
```

Comment:
`kubernetesVersion` will actually change in the different ci tests.

See line 111:
https://github.com/Azure/karpenter-provider-azure/blob/5679a16276876a86af042290a4f857e7527c92d5/pkg/providers/imagefamily/nodeimage_test.go#L111